scripts/SANS/sans/gui_logic/presenter/property_manager_service.py

```diff
@@ -0,0 +1,107 @@
+from mantid.api import Algorithm
+from mantid.kernel import (PropertyManagerProperty, PropertyManagerDataService)
+from sans.state.state_base import create_deserialized_sans_state_from_property_manager
+
+
+class PropertyManagerService(object):
+    sans_property_manager_prefix = "SANS_PROPERTY_MANAGER_THIS_NEEDS_TO_BE_UNIQUE_"
+
+    class ConverterAlgorithm(Algorithm):
```

Comment:
I see. Could we not just use the `PropertyManagerProperty` directly rather than needing to go through an algorithm?